modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -584,9 +584,21 @@ Third party password managers can also be integrated. The only requirement is th
 password in plain text somehow. If not, additional operations might be required to get the password as
 plain text and inject it in the listener.
 
-As an example:
+Example using "Vault" as secrets store
+
+* See https://learn.hashicorp.com/collections/vault/getting-started
```

Comment:
can you make this link a macro as we do with the other links?
see on top of the page
like: `:vault-getting-started-url: https://learn.hashicorp.com/collections/vault/getting-started`
and here: `See the {vault-getting-started-url}[get started here guide]